sdk/core/azure-core/azure/core/pipeline/transport/_bigger_block_size_http_adapters.py

```diff
@@ -41,5 +41,7 @@ def get_connection(self, url, proxies=None):
         conn = super(BiggerBlockSizeHTTPAdapter, self).get_connection(url, proxies)
         system_version = tuple(sys.version_info)[:3]
         if system_version[:2] >= (3, 7):
+            if not conn.conn_kw:
+                conne.conn_kw = {}
```

Comment:
Actually, `conn.conn_kw` will never be `None` as it is created from a function kwargs:

https://github.com/urllib3/urllib3/blob/67111020d9cfc5798bc85f3719b3015e3ff28692/src/urllib3/connectionpool.py#L167

```py
    def __init__(
        self,
        ...
        **conn_kw,
```

and saved as an attribute later:

https://github.com/urllib3/urllib3/blob/67111020d9cfc5798bc85f3719b3015e3ff28692/src/urllib3/connectionpool.py#L195

```py
        self.conn_kw = conn_kw
```